### DISCUSSION TITLE
How to change the direction of self loop?

### DISCUSSION THREAD

**Discussion by GongSiqiu** [networkx/networkx#5429] [Q&A] [answered]
![图片](https://user-images.githubusercontent.com/40697840/160244872-0a61c69b-db53-44e3-8fd4-b9b12e06b98f.png)
All self loops are upside. Is it possible to change the direction to make it look symmetrically

**Comment by rossbar** [comment, marked as answer]
Unfortunately this isn't currently supported. You might be able to modify the `FancyArrowPatch`es directly, or perhaps there's a way to improve things with a specific connectionstyle, but there's currently no straightforward way to specify the orientation of selfloops.
